Makefile

```
@@ -544,6 +544,10 @@ ray-project-mini-image-build:
 		$(IMAGE_BUILD_EXTRA_OPTS) \
 		-f ./hack/testing/ray-mini/Dockerfile ./ \
 
+.PHONY: ray-project-mini-image-build-push
+kind-ray-project-mini-image-build: PUSH=--push
+kind-ray-project-mini-image-build: ray-project-mini-image-build
```

checkmake (0.3.2)

[warning] Target "kind-ray-project-mini-image-build" defined multiple times (lines 548 and 549).

(uniquetargets)